src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -317,9 +317,9 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None, input_meta=None):
                 # Dataset object no longer contains untrimmed data
                 data, meta = util.trim(data, meta)
 
-                # Create bad pixel mask (1 = good, 0 = bad)
+                # Create bad pixel mask (False = good, True = bad)
                 data['mask'] = (['time', 'y', 'x'],
-                                np.ones(data.flux.shape, dtype=bool))
+                                np.zeros(data.flux.shape, dtype=bool))
```

Comment:
`util.trim()` should still work totally fine since the `data.isel` command works on all DataArrays in the Dataset, and the mask doesn't use any explicit integers or anything that would depend on the pixel indices